pandas/tests/test_strings.py

```diff
@@ -659,6 +661,31 @@ def test_ismethods(self):
         self.assertEquals(str_s.str.isupper().tolist(), [v.isupper() for v in values])
         self.assertEquals(str_s.str.istitle().tolist(), [v.istitle() for v in values])
 
+    def test_isnumeric(self):
+        # 0x00bc: ¼ VULGAR FRACTION ONE QUARTER
+        # 0x2605: ★ not number
+        # 0x1378: ፸ ETHIOPIC NUMBER SEVENTY
+        # 0xFF13: ３ Em 3
+        values = ['A', '3', unichr(0x00bc), unichr(0x2605),
```

Comment:
I think we can't use `u( )` here because escaped unicode is not handled as numeric or decimal.